src/material/core/common-behaviors/common-module.ts

```diff
@@ -57,42 +57,37 @@ export class MatCommonModule {
   /** Whether we've done the global sanity checks (e.g. a theme is loaded, there is a doctype). */
   private _hasDoneGlobalChecks = false;
 
-  /** Configured sanity checks. */
-  private _sanityChecks: SanityChecks;
-
-  /** Used to reference correct document/window */
-  protected _document: Document;
-
   constructor(
     highContrastModeDetector: HighContrastModeDetector,
-    @Optional() @Inject(MATERIAL_SANITY_CHECKS) sanityChecks: any,
-    @Inject(DOCUMENT) document: any,
+    @Optional() @Inject(MATERIAL_SANITY_CHECKS) private _sanityChecks: SanityChecks,
+    @Inject(DOCUMENT) private _document: Document,
```

Comment:
Does the new package format eliminate the problem with typing constructor params with browser global types like `Document`?

Reply:
I think that it was more of a ViewEngine vs Ivy issue than with packaging. As long as we're using `@Inject` we should be fine.